### ISSUE TITLE
Bug: Default footer has tabbing issues

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#1572]
# Info

If you use the default `<goa-app-footer>` with no other code inside, it outputs a copyright notice on the left and the alberta logo on the right. But the tab order doesn't go from left to right, it goes from right to left. If you add a meta-section in the footer, then the tab order is respected correctly

## Acceptance Criteria:

1. Tab order should be from left to right in default app-footer

**Comment by ArakTaiRoth** [comment]
@chrisolsen If you have a quick fix, let me know and I can inform the team
